open_catalyst_dataset/ase_dbs/__init__.py

```diff
@@ -2,3 +2,4 @@
 
 
 BULK_DB = os.path.join(__path__[0], 'bulks.db')
+adsorbate_DB = os.path.join(__path__[0], 'adsorbates.db')
```

Comment:
Objects like this defined outside of functions are constants. By PEP conventions, constants should be capitalized.